tests/functional/test_xdd_load_file.sh

```diff
@@ -0,0 +1,128 @@
+#!/usr/bin/env bash
+#
+# Description - test xdd seek load
+#
+# Validate the functionality of xdd seek load is able to load a file with or without
+# numreqs. It should also overwrite the blocksize and reqsize if provided.
+#
+# Get absolute path to script
+SCRIPT=${BASH_SOURCE[0]}
+SCRIPTPATH=$(dirname "${SCRIPT}")
+
+# Source the test configuration environment
+source "${SCRIPTPATH}/../test_config"
+source "${SCRIPTPATH}/../common.sh"
+
+# Perform pre-test
+initialize_test
+test_dir="${XDDTEST_LOCAL_MOUNT}/${TESTNAME}"
+
+# Function to load a file with optional numreqs, blocksize, and reqsize
+seek_load_file() {
+  local load_file=$1
+  local numreqs=$2
+  local blocksize=$3
+  local reqsize=$4
+  local cmd="${XDDTEST_XDD_EXE} -seek load ${load_file} -target /dev/null"
+
+  # If the optional parameters are provided, add them to the command
+  if [[ -n "${numreqs}" ]]; then
+    cmd+=" -numreqs ${numreqs}"
+  fi
+  if [[ -n "${blocksize}" ]]; then
+    cmd+=" -blocksize ${blocksize}"
+  fi
+  if [[ -n "${reqsize}" ]]; then
+    cmd+=" -reqsize ${reqsize}"
+  fi
+
+  # Execute the command
+  eval "${cmd}"
+}
+
+# Function to verify the output
+verify_output() {
+  local output=$1
+  local expected_blocksize=$2
+  local expected_reqsize=$3
+  local expected_numreqs=$4
+
+  # Extract the blocksize, reqsize, and numreqs from the output
+   local blocksize
```

Comment:
Extra space

Reply:
Also, `local` is not necessary.

Reply:
it's necessary because when the test ran, it throws the error below:
```
In ./tests/functional/test_xdd_load_file.sh line 19:
log_file="$(get_log_file)"
^------^ SC2034 (warning): log_file appears unused. Verify use (or export if used externally).


In ./tests/functional/test_xdd_load_file.sh line 52:
  local blocksize=$(echo "$output" | grep -oP 'Blocksize in bytes, \K[0-[9](https://github.com/MigeljanImeri/xdd/actions/runs/11420229589/job/31775685737#step:5:10)]+')
        ^-------^ SC2155 (warning): Declare and assign separately to avoid masking return values.


In ./tests/functional/test_xdd_load_file.sh line 53:
  local reqsize=$(echo "$output" | grep -oP 'Request size, [0-9]+, [10](https://github.com/MigeljanImeri/xdd/actions/runs/11420229589/job/31775685737#step:5:11)24-byte blocks, \K[0-9]+')
        ^-----^ SC2155 (warning): Declare and assign separately to avoid masking return values.


In ./tests/functional/test_xdd_load_file.sh line 54:
  local numreqs=$(echo "$output" | grep -oP 'Number of Operations, \K[0-9]+')
        ^-----^ SC2[15](https://github.com/MigeljanImeri/xdd/actions/runs/11420229589/job/31775685737#step:5:16)5 (warning): Declare and assign separately to avoid masking return values.

For more information:
  https://www.shellcheck.net/wiki/SC2034 -- log_file appears unused. Verify u...
  https://www.shellcheck.net/wiki/SC2155 -- Declare and assign separately to ...
```

Reply:
Weird. Ok.